bgpd/bgp_evpn_vty.c

```diff
@@ -3703,7 +3703,7 @@ DEFUN(show_bgp_l2vpn_evpn_route_rd,
 	bool uj = false;
 	json_object *json = NULL;
 
-	bgp = bgp_get_default();
+	bgp = bgp_get_evpn();
```

Comment:
Changing this to bgp_evpn would be better. I guess you had already followed this in few other PRs.

Reply:
I only did when the name was in the like of `bgp_default`, i.e. where the meaning was wrong. I am not sure about the simple `bgp` name though.